Dockerfile.IntelVSP.rhel

```diff
@@ -0,0 +1,24 @@
+#/usr/local/lib/python3.9/site-packages/ SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2024 Intel Corporation
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.22-openshift-4.18 AS builder
+ARG TARGETOS
+ARG TARGETARCH
+
+COPY . /usr/src/ipu-opi-plugin
+WORKDIR /usr/src/ipu-opi-plugin
+RUN CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -o bin/ipuplugin cmd/intelvsp/intelvsp.go
+
+FROM registry.ci.openshift.org/ocp/4.18:base-rhel9
+
+COPY --from=builder /usr/src/ipu-opi-plugin/bin/ipuplugin /usr/bin/
+COPY cmd/intelvsp/fxp-net_linux-networking.pkg /
```

Comment:
i.e. https://github.com/bn222/cluster-deployment-automation/blob/main/dpuVendor.py#L71

Reply:
We can't have that as variables at this level, hence I've replaced the vars with what we are using.

Also, since there is code that reads the env var (which we should remove, and use a proper function to set it), I've added it to the call-site.